plugins/org.python.pydev.core/pysrc/_pydev_runfiles/pydev_runfiles_unittest.py

```diff
@@ -87,7 +88,10 @@ def stopTest(self, test):
 
 
         diff_time = '%.2f' % (end_time - self.start_time)
-        if not self._current_errors_stack and not self._current_failures_stack:
+        if self._test_skipped:
```

Comment:
I took a quick look under how to get if the test was skipped. It seems that the `test._outcome` has it, so, instead of trying to put it in the class, you can do something as:

```
        skipped = False
        outcome = getattr(test, '_outcome', None)
        if outcome is not None:
            skipped = bool(getattr(outcome, 'skipped', None)
```

Note: using `getattr` with default values to be backward/forward compatible so that this won't raise exceptions if that changes (or if it was different in an older version of Python/Jython).

Reply:
Thanks for the feedback. Getting the "skipped" info was my main problem.
I do not see any _outcome or similar in the test data.
Where did you see that?

In results is a list called skipped, which accumulates all skipped testcases.

My idea was to add a similar method to addSuccess, addError and addFailure in the case to mark the specific testcase.

Reply:
Which version of Python are you using? I can see `test._outcome` in Python 3.8 (you can see where it's set in the CPython standard library: https://github.com/python/cpython/blob/6b34d7b51e33fcb21b8827d927474ce9ed1f605c/Lib/unittest/case.py#L585).

Reply:
Damn you are completely right.
This was the problem, I am currently stuck on 2.7 - but we are migrating atm.

I will change to 3.8,implement it as you suggested and repush.

Thanks a lot for your help!

Reply:
@herostrat Do you plan to do this in the short term? (if not, I'll close the pull request)

Reply:
Yes, sorry for the delay. I will push at latest on friday.